.github/workflows/ci_test-conda.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8"]
-        pytorch-version: ["1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11"]
+        pytorch-version: ["1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11", "1.12"]
```

Comment:
Do we really need to test all these versions? That's quite exhaustive

Reply:
they are running in parallel so I think it is fine...

Reply:
not completely. We have a limited number of runners. So if we occupy them here, some other actions do have to wait.